Luminary099/R30.agc

```diff
@@ -291,15 +292,15 @@ NEXTLINE	STOVL	RONE		# RN AT (-29)M FOR EARTH OR MOON
 		BON	GOTO
```

Comment:
![image](https://user-images.githubusercontent.com/3440094/95389284-7f3d6980-08eb-11eb-8051-eb4823dc6285.png)
5 lines above:

- Extra 2 spaces before `VN FETCH`
- Space between `RN` and `,` (comma)